espnet2/bin/st_inference.py

```diff
@@ -306,7 +306,7 @@ def inference(
     token_type: Optional[str],
     bpemodel: Optional[str],
     allow_variable_data_keys: bool,
-    enh_s2t_task: bool,
+    enh_s2t_task: bool = False,
```

Comment:
@simpleoier, is it OK?

Reply:
@D-Keqi, can you tell me why you specify it `False`?

Reply:
Sorry @sw005320, it's my fault. I missed one argument definition in the pull request. I put it in the [asr_inference](https://github.com/espnet/espnet/blob/40202fca64529982fa7c1c428137243f306a0d5d/espnet2/bin/asr_inference.py#L553), but not in the st_inference. @D-Keqi Can you wait for my fix or fix it in this PR?

Reply:
I submitted a PR #4323. If @D-Keqi fix it in this PR, I can close the other one.

Reply:
@simpleoier thanks! I think based on your PR, my modification to the st_inference.py is not needed.